### PR TITLE
bug 957802, 1286541: Documentation updates

### PR DESCRIPTION
**Pull request by jwhitlock**
* Update CKEditor documentation with current process
* Switch to readthedocs.io
* Document the iperceptions waffle flag

r? @stephaniehobson

### CODE REVIEW

**Comment by jwhitlock** [comment]
Force-push to rebase on master and document PR #3938.

**Comment by jwhitlock** [comment]
And another force-push for PR #3932 fixes.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,14 +67,14 @@ If you have questions about what to work on, you can ask:
 How to submit code
 ==================
 
-MDN development process is very much like [these Mozilla Webdev guidelines](http://mozweb.readthedocs.org/en/latest/guide/development_process.html).
+MDN development process is very much like [these Mozilla Webdev guidelines](https://mozweb.readthedocs.io/en/latest/guide/development_process.html).
```
Comment (stephaniehobson) [inline CONTRIBUTING.md]: 🔒👍

File: docs/ckeditor.rst [hunk 2]
```diff
@@ -9,19 +9,25 @@ document details how to update CKEditor within the MDN codebase.
 
 Building CKEditor
 -----------------
+To rebuild CKEditor::
 
-Building CKEditor is quite easy!
```
Comment (stephaniehobson) [inline docs/ckeditor.rst]: ~~is quite easy~~ 👍

**Comment by stephaniehobson** [comment]
R 👍

**Pull request merged**